testgrid/config.yaml

```diff
@@ -3110,7 +3110,12 @@ test_groups:
   alert_stale_results_hours: 168
   num_failures_to_alert: 1
 
+- name: pull-k8s-cluster-bundle-bazel-test
+  gcs_prefix: k8s-cluster-bundle/logs/pull-k8s-cluster-bundle-bazel-test
+- name: post-k8s-cluster-bundle-bazel-test
+  gcs_prefix: k8s-cluster-bundle/logs/post-k8s-cluster-bundle-bazel-test
```

Comment:
`Job post-k8s-cluster-bundle-bazel-test does not have a matching testgrid testgroup`

I think the `gcs_prefix` fields are wrong? @krzyzacy

Reply:
oops :-) it should be 

presubmit: `kubernetes-jenkins/pr-logs/directory/pull-k8s-cluster-bundle-bazel-test`
postsubmit/periodic: `kubernetes-jenkins/logs/post-k8s-cluster-bundle-bazel-test`

Reply:
the ones above (`k8s-conformance-gardener`) is not a prow job :jack_o_lantern:

Reply:
Is an explanation of the rules/best practices about this file documented anywhere?

Reply:
@cjwagner we probably should document how we map presubmit jobs to bucket/pr-logs in podutils

for bootstrap you explicitly supply upload target like https://github.com/kubernetes/test-infra/blob/master/config/jobs/GoogleCloudPlatform/k8s-multicluster-ingress/k8s-multicluster-ingress-config.yaml#L15

Reply:
Bootstrap doesn't just use the path provided by the `upload` flag. It appends to the path based on the org, repo, and PR number if applicable. It also has some other surprising behavior like removing the org (and repo?) if the value is "kubernetes".

The podutils provide support for multiple different upload strategies, one of which is the `legacy` strategy that matches bootstrap. https://github.com/kubernetes/test-infra/blob/fd03336ae3249d070d28c3f550c9abb313d64ee3/prow/apis/prowjobs/v1/types.go#L196-L202

@stevekuznetsov Is this correct? Is there any documentation for the different strategies or how to configure them?